dev/tasks/tasks.yml

```
@@ -1766,6 +1766,16 @@ tasks:
       r_tag: 3.6-centos8
       not_cran: "TRUE"
 
+  test-r-rstudio-r-base-3.6-centos7:
```

Just to be explicit which version this is

```suggestion
  test-r-rstudio-r-base-3.6-centos7-devtoolset-8:
```